src/__snapshots__/synchronizeContacts.spec.js.snap

```diff
@@ -13,67 +17,155 @@ Array [
-      "createdAt": "2018-05-05T09:09:00.115Z",
-      "createdByApp": "konnector-google",
+      "createdAt": "2016-06-30T09:33:00.123Z",
+      "createdByApp": "Contacts",
       "createdByAppVersion": "2.0.0",
       "doctypeVersion": 2,
-      "importedAt": "2018-05-05T09:09:00.115Z",
+      "importedAt": "2016-06-30T09:33:00.123Z",
       "importedFrom": "konnector-google",
+      "sourceAccount": "45c49c15-4b00-48e8-8bfd-29f8177b89ff",
       "sync": Object {
         "45c49c15-4b00-48e8-8bfd-29f8177b89ff": Object {
           "contactsAccountsId": "45c49c15-4b00-48e8-8bfd-29f8177b89ff",
-          "id": "people/78485",
+          "id": "people/987654",
           "konnector": "konnector-google",
           "lastSync": "2018-05-05T09:09:00.115Z",
-          "remoteRev": "etag-6deab8dd-5a0f-451d-bcfe-fd81dd57bdcd",
+          "remoteRev": "6020dd2f-c9b8-4865-bdbd-078faad65204",
         },
       },
       "updatedAt": "2018-05-05T09:09:00.115Z",
       "updatedByApps": Array [
+        "Contacts",
         "konnector-google",
       ],
     },
-    "id": "larue-cremin-attached-to-another-source",
+    "email": Array [
+      Object {
+        "address": "john.doe@posteo.net",
+        "primary": false,
+        "type": "personal",
+      },
+      Object {
+        "address": "john.doe@cozycloud.cc",
+        "primary": true,
+      },
+    ],
+    "id": "john-doe-edited",
+    "metadata": Object {
+      "google": Object {
+        "metadata": Object {
+          "deleted": false,
+        },
+      },
+      "version": 1,
+    },
     "name": Object {
-      "familyName": "Cremin",
-      "givenName": "Larue",
+      "familyName": "Doe",
+      "givenName": "John",
     },
+    "note": null,
+    "phone": Array [],
+    "vendorId": "people/987654",
   },
 ]
 `;
 
-exports[`synchronizeContacts synchronize contacts with google to cozy strategy should synchronize cozy contacts from google contacts 1`] = `
+exports[`synchronizeContacts function should synchronize contacts: johnDoeInGoogle 1`] = `
+Array [
+  Object {
+    "addresses": Array [],
+    "biographies": null,
+    "birthdays": Array [
+      Object {
+        "date": Object {
+          "day": "22",
+          "month": "2",
+          "year": "1955",
+        },
+        "metadata": Object {
+          "primary": true,
+        },
+      },
+    ],
+    "emailAddresses": Array [
+      Object {
+        "metadata": Object {
+          "primary": false,
+        },
+        "type": "personal",
+        "value": "john.doe@posteo.net",
+      },
+      Object {
+        "metadata": Object {
+          "primary": true,
+        },
+        "type": undefined,
+        "value": "john.doe@cozycloud.cc",
+      },
+    ],
+    "names": Array [
+      Object {
+        "familyName": "Doe",
+        "givenName": "John",
+      },
+    ],
+    "organizations": Array [
+      Object {
+        "name": "Cozy cloud",
+      },
+    ],
+    "phoneNumbers": Array [],
+  },
+]
+`;
```

Comment:
Is is wanted that some contacts don't have : 
- "biographies": null,
or 
- note : null ?

Reply:
every contact written in google should have `biographies` and every contact written in cozy should have `note` (just checked the snapshots, it is the case)